pkgs/tools/networking/debian-hostname/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, lib, fetchurl }:
+
+stdenv.mkDerivation rec {
+  pname = "debian-hostname";
```

Comment:
Personally, I'd call the package `hostname-debian`. Hostname is more important than Debian so it should come first.